docs/end-user-guide/using-the-plugin.md

```diff
@@ -0,0 +1,11 @@
+## Connecting an account
+
+Use the `/connect` slash command to connect.
+
+## Starting a call
+
+Start a call either by clicking the header button or the `/start` slash command. Every meeting you create will create a new meeting room in MS Teams. If two meetings are created really close (less than 30 seconds) you will get a prompt asking you whether you are sure you want to create the meeting.
+
+## Disconnecting an account
+
+Use the `/disconnect` slash command to disconnect the account.
```

Comment:
```suggestion
Use the `/mstmeetings disconnect` slash command to disconnect the account.
```

Reply:
I think it is not needed to give so much information on what disconnect can be used for. I think only the first paragraph is enough.